defaults/test_default.yaml

```diff
@@ -48,7 +48,7 @@ seeds_num: 1
 instance_provision: "spot"
 
 execute_post_behavior: false
-post_behavior_db_nodes: "keep-on-failure"
+post_behavior_db_nodes: "destroy"
```

Comment:
I'm not sure this is enough, we need to change the longevity pipeline defaults

Reply:
i think it doesn't have defaults

Reply:
@fruch , fixed

Reply:
I think both post_behavior_monitor_nodes, post_behavior_k8s_cluster, should be changed as well

Reply:
done